src/main/java/Event.java

```diff
@@ -0,0 +1,22 @@
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class Event extends Task{
+
+    protected String at;
```

Comment:
Maybe you can change the variable name from "at" to "eventTime" so that it sounds more meaningful?